CONTRIBUTING.md

```diff
@@ -105,7 +105,7 @@ If you'd like to test latest Kaoto UI and not rely on a released version, follow
   * `yarn build:lib`
 * Open VS Code on `vscode-kaoto` local clone folder
 * `yarn`
-* `yarn link` _\<kaoto-ui local clone folder uri>_
+* `yarn link` _\<kaoto-ui local clone folder uri>/packages/kaoto-ui_
   * i.e. `yarn link ~/repositories/kaoto-ui`
```

Comment:
how should example on this line looks like? `yarn link ~/repositories/kaoto-ui/packages/kaoto-ui_` is this correct?

Reply:
updated :+1: